source/contributed/ps_ubuntu.md

```diff
@@ -104,7 +104,7 @@ During this step you will need a Steam API Key, which you can obtain [from here]
 mkdir ~/world
 cd ~/world
 npm install screeps
-./node_modules/screeps/bin/screeps.js init
+npx screeps init
```

Comment:
You should make sure that `npx` is actually installed, so it might be a good idea to install it here as well.